### ISSUE TITLE
Exception when adding an object to a GraphicsLayer created in code, but only in WASM

### ISSUE THREAD

**Issue by anotherlab** [dymaptic/GeoBlazor#256]
If I create a GraphicsLayer in code for Blazor WASM, I get a `System.InvalidOperationException` exception when I add anything to the layer. The exception message is `The render handle is not yet assigned`.  In MAUI Hybrid Blazor, the code works as expected.

I create the layer like this in the `OnViewRendered` event of the `MapView`
```csharp
GraphicsLayer? customLayer;
Task ViewRendered()
{
    if (customLayer == null)
    {
        customLayer = new GraphicsLayer(null, "Code Behind", 1.0, true, ListMode.Show);
        // Using the this syntax produces the same results
        customLayer = new GraphicsLayer(null, "Code Behind", null, null, null);
        return _mapView!.AddLayer(customLayer);
    }
    return Task.CompletedTask;
}
```
I am adding a marker for a point like this:

```csharp
SimpleMarkerSymbol symbol = new SimpleMarkerSymbol(new Outline(new MapColor("blue")), new MapColor("red"), 10);

var marker = new Graphic(new Point(point.Longitude, point.Latitude), symbol);

await customLayer!.Add(marker);
```
The exception happens at the customLayer!Add(marker) line. Is this a bug or am I defining the layer incorrectly?

# What is happening ##
A `System.InvalidOperationException` of `The render handle is not yet assigned.` is thrown
## What should happen ##
A marker symbol should be displayed

# Steps to reproduce the problem #
1. Clone the [demo](https://github.com/anotherlab/GeoBlazorCustomLayerException)
2. Restore the nuget packages
3. Edit [Program.cs](https://github.com/anotherlab/GeoBlazorCustomLayerException/blob/main/WasmCreateGraphicsLayers/Program.cs#L13) and replace "INSERT YOUR KEY HERE" with your API key
4. Build and the run app
5. You should see a map. There will be a layerlist widget with 3 layers listed. The one labeled "Code Behind" is created in the code, and the other two are in the markup
6. Click the "Draw Route" button. You will see a route and stops drawn on the markup layers. 
7. The stops should also be drawn on the "Code Behind" but are not and an exception message will display below the map

Tested under Windows 11 with Blazor web as the target platform. Compiled with VS 2022 17.8.0 Preview 4.0 and GeoBlazor 2.4.0

**Comment by AndersenBell** [comment]
Thank you for the project example, it was very easy to replicate the issue. It looks like this problem has been solved in the latest release of GeoBlazor 2.5.0 (in both .net 7 and .net 8) can you try updating your project to the latest version and confirming if it's fixed for you?

**Comment by TimPurdum** [comment]
Closing as we believe this is fixed.
